packages/contracts-rfq/contracts/FastBridgeV2.sol

```diff
@@ -139,8 +139,13 @@ contract FastBridgeV2 is Admin, IFastBridgeV2 {
         if (block.timestamp > transaction.deadline) revert DeadlineExceeded();
 
         // mark bridge transaction as relayed
-        if (bridgeRelays[transactionId]) revert TransactionRelayed();
-        bridgeRelays[transactionId] = true;
+        if (bridgeRelayDetails[transactionId].blockNumber != 0) revert TransactionRelayed();
```

Comment:
To avoid repetition, let's make `bridgeRelays(bytes32)` a public view - will make it easier to maintain later on.